docs/data-sources/datalabel.md

```diff
@@ -1,6 +1,6 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "cyral_datalabel Data Source - cyral"
+page_title: "cyral_datalabel Data Source - terraform-provider-cyral"
```

Comment:
Could you please also update the docker-compose.yaml file to use terraform-provider-cyral as the root for the project, something like:
```docker
  app:
    image: golang:1.19
    container_name: terraform_provider_cyral
    volumes:
      - .:/go/src/terraform-provider-cyral
    working_dir: /go/src/terraform-provider-cyral
```
This will make sure that no unnecessary diffs are generated in the future, regardless if one is running `tfplugindocs` from docker or locally.